3696/3/app/controllers/user_controller.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require_relative '../helpers/auth_helper'
+require_relative '../helpers/info_helper'
+
+class UserController < Sinatra::Base
```

Comment:
Usually, developers call it SessionsController, because UsersController handles interactions with users